R/shiny.R

```diff
@@ -1348,10 +1348,11 @@ ShinySession <- R6Class(
     registerDataObj = function(name, data, filterFunc) {
       # abusing downloads at the moment
       self$downloads$set(name, list(data = data, filter = filterFunc))
-      return(sprintf('session/%s/dataobj/%s?w=%s',
+      return(sprintf('session/%s/dataobj/%s?w=%s?timestamp=%s',
```

Comment:
Oops, sorry, changed it to `&`. I actually prefer the `createUniqueId` too, it looks much neater than `as.character(as.numeric(Sys.time()))`. Good to merge?